service/history/transferQueueTaskExecutorBase.go

```diff
@@ -190,7 +190,7 @@ func (t *transferQueueTaskExecutorBase) recordWorkflowStarted(
 		WorkflowTypeName:   workflowTypeName,
 		StartTimestamp:     startTimeUnixNano,
 		ExecutionTimestamp: executionTimeUnixNano,
-		RunTimeout:         int64(runTimeout),
+		RunTimeout:         int64(timestamp.DurationValue(runTimeout).Round(time.Second).Seconds()),
```

Comment:
I expect all these rounds to be temporary, right?

Reply:
Some of these will stay due to Cassandra TTL being in seconds.